docs/book/src/topics/getting-started.md

```diff
@@ -80,20 +80,27 @@ For example, if your password is `foo'blah$`, you should do `export AZURE_CLIENT
 
 </aside>
 
-<aside class="note warning"> 
+<aside class="note warning">
 
-<h1> Warning </h1> 
+<h1> Warning </h1>
 
-The capability to set credentials using environment variables is now deprecated and will be removed in future releases, the recommended approach is to use `AzureClusterIdentity` as explained [here](multitenancy.md) 
+The capability to set credentials using environment variables is now deprecated and will be removed in future releases, the recommended approach is to use `AzureClusterIdentity` as explained [here](multitenancy.md)
 
 </aside>
 
 
 ### Building your first cluster
+In case you are not looking to install additional ASO CRDs, set `ADDITIONAL_ASO_CRDS` to empty string. i.e.
+```bash
+export ADDITIONAL_ASO_CRDS=""
```

Comment:
Is there no way to avoid this? We don't want to have to bother a new user with ASO details....

Reply:
I added it as a guardrail but it is not necessary to mention. The Kustomize substitution we added at https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4547/files#diff-e7e1b6491263f40fd6772309861b747b8800e4b9c3c8a73abaf7f2630cd059d9R28 takes care of _unset_ values. 

Will update the documentation.